test/apt_cache.bats

```diff
@@ -40,6 +40,11 @@ setup() {
     gpg --verify ${FREIGHT_CACHE}/dists/example/Release.gpg ${FREIGHT_CACHE}/dists/example/Release
 }
 
+@test "freight-cache works without tty" {
+    freight_cache_nohup -v
+    assert_success
```

Comment:
Or just remove the line as there are no other assertions. The line above will exit on error.